.github/workflows/go.yml

```diff
@@ -1,13 +1,40 @@
 name: Go
 on: [push]
 jobs:
+    build:
+        name: Build
+        runs-on: ubuntu-latest
+        steps:
 
-  build:
-    name: GoCity CI
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@master
-      - uses: actions/setup-go@v1
-        with:
-          version: '1.12'
-      - run: go test -race -cover ./...
+            - name: Set up Go 1.12
+              uses: actions/setup-go@v1
+              with:
+                  go-version: 1.12.4
+
+            - name: Check out code into the Go module directory
+              uses: actions/checkout@v1
+              with:
+                  fetch-depth: 1
+                  path: go/src/github.com/user/gocity
+
+            - name: GoLint
+              run: |
+                  go get golang.org/x/lint/golint
```

Comment:
I recommend you use https://github.com/golangci/golangci-lint. There are a lot of linters, including golint package.

Reply:
GolangCI-Lint seems much better and provides mulitple linters in parallel, so I have changed the linter to it. I have also set the flag to return error code 0, so the github action doesn't fail.